build_cmake/CMakeLists.txt

```diff
@@ -24,7 +24,7 @@ set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
 if (NOT TARGET Framework)
-	add_subdirectory(../../Framework/build_cmake ${CodeGen_SOURCE_DIR}/../../Framework/build_cmake/build)
+	add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/../../Framework/build_cmake ${CMAKE_CURRENT_SOURCE_DIR}/../../Framework/build_cmake/build)
```

Comment:
The linked TargetArch cmake module doesn't seem to identify arm64 targets, which is quite a shame. 

Anyways, I think it's only a problem on Android, and if the idea is to use the CMake gradle plug-in (which seems to handle multiple arch builds), it might not be much of an issue.